org.mixedrealitytoolkit.core/Interactables/StatefulInteractable.cs

```diff
@@ -177,6 +177,12 @@ public string SpeechRecognitionKeyword
         [field: SerializeField, EditableTimedFlag, FormerlySerializedAs("isToggled"), Tooltip("Is the interactable toggled?")]
         public TimedFlag IsToggled { get; private set; } = new TimedFlag();
 
+        /// <summary>
+        /// Is this object proximity-hovered by an active Collider + Interactor combination?
+        /// </summary>
+        [field: SerializeField, EditableTimedFlag, Tooltip("Is this object proximity-hovered by an active Collider + Interactor combination?")]
+        public TimedFlag IsProximityHovered { get; private set; } = new TimedFlag();
```

Comment:
Let's move IsProximityHovered in the PressableButton.

I'm not sure it makes sense to be in StatefuleIteractable, since nothing in StatefuleIteractable will change this value.

Reply:
That was my original approach too but since TimedFlag resides in MixedReality.Toolkit.Core binaries and its Active property is of type 'internal' I cannot change it if I declare it in PressableButton because that class exists in MixedReality.Toolkit.UXCore binaries and don't have access to 'internal' properties of other binaries.

Reply:
Moved IsProximityHover property to PressableButton class and MixedReality.Toolkit.Core internals visible to MixedReality.Toolkit.UX assembly in [latest commit](https://github.com/MixedRealityToolkit/MixedRealityToolkit-Unity/pull/611/commits/7a763b5a32c2b87e5f95aba1c4c147eead377387).